content/python/concepts/built-in-functions/terms/sorted/sorted.md

```diff
@@ -115,3 +115,30 @@ my_sorted_list = sorted(my_list, key=sorting_func)
 print(my_sorted_list)
 # Output: [4, 3, 2, 1]
 ```
+
+## Example 6
+
+Sorting a list of dictionaries based on a specific key:
+
+Given a list of dictionaries, they can be sorted based on a specific key within each dictionary: 
+
+```python
+my_list_of_dicts = [
+    {"name": "Alice", "age": 30},
+    {"name": "Bob", "age": 25},
+    {"name": "Charlie", "age": 35},
+    {"name": "David", "age": 28}
+]
+
+# Sort the list of dictionaries based on the "age" key in ascending order
+sorted_list = sorted(my_list_of_dicts, key=lambda x: x["age"])
+
+print(sorted_list)
+# Output: [{'name': 'Bob', 'age': 25}, {'name': 'David', 'age': 28}, {'name': 'Alice', 'age': 30}, {'name': 'Charlie', 'age': 35}]
```

Comment:
```suggestion
```

Reply:
codebytes are runnable so the output will be displayed, it doesn't need to be included as a comment